deploy/charts/cert-manager/templates/cainjector-deployment.yaml

```diff
@@ -16,7 +16,7 @@ metadata:
   {{- end }}
 spec:
   replicas: {{ .Values.cainjector.replicaCount }}
-  {{- if ne (quote .Values.global.revisionHistoryLimit) (quote "") }}
+  {{- if not (has (quote .Values.global.revisionHistoryLimit) (list "" (quote ""))) }}
```

Comment:
Please add a comment above all these explaining why it is written this way and link to the upstream bugs.